cli/src/ui.rs

```diff
@@ -45,7 +46,7 @@ enum UiOutput {
 
 /// A builtin pager
 pub struct BuiltinPager {
-    pager: MinusPager,
+    pager: Arc<RwLock<Option<MinusPager>>>,
```

Comment:
I'd add a comment:

> `Option<MinusPager>` is set to None after the pager quits (e.g. the user quits the pager early).

Reply:
Agreed! This is one of the things I was hoping to do before taking this out of draft. 🙂 Will attend to this soon.